source/common/router/router.cc

```diff
@@ -31,13 +31,16 @@
 #include "common/router/debug_config.h"
 #include "common/router/retry_state_impl.h"
 #include "common/runtime/runtime_impl.h"
+#include "common/stream_info/uint32_accessor_impl.h"
 #include "common/tracing/http_tracer_impl.h"
 
 #include "extensions/filters/http/well_known_names.h"
 
 namespace Envoy {
 namespace Router {
 namespace {
+constexpr char NumPreviousInternalRedirectFilterStateName[] = "num_previous_internal_redirect";
```

Comment:
I think this could be num_internal_redirects as well, since we increment it as we do the redirect.

Reply:
Done.